src/search-schema/data/definitions/enums/assay_types.yaml

```diff
@@ -64,23 +64,41 @@ image_pyramid:
     contains-pii: false
     vitessce-hints: ['is_image', 'pyramid']
     
-IMC:
-    description: Imaging Mass Cytometry
-    alt-names: []
+IMC2D:
+    description: 2D Imaging Mass Cytometry
```

Comment:
```suggestion
    description: Imaging Mass Cytometry (2D)
```

Reply:
Concur- I just used the other order because it was more common in our Slack conversations.  I agree your order is preferable.